### DISCUSSION TITLE
Laser engraving problem

### DISCUSSION THREAD

**Discussion by MArTI9999** [arkypita/LaserGRBL#1810] [Q&A]
Hello.
I made my laser engraver with arduino,cnc shield and chinese laser but I have a problem.
Laser engraving really dark or not at all.It looks like laser started burning when hit about 40% even if Smin=0 Smax=1000.Speed doesn't matter(image is just brighter or darker).The grayscale is narrow.
I found similar problem but I don't know how to fix it in my case.
https://forum.lightburnsoftware.com/t/power-modulation-problem-i-think/10483

Is it possible to adjust the configuration or do I have to change the laser?

![Bez tytułu](https://user-images.githubusercontent.com/103198639/162238946-d98d9766-9262-4bbe-a623-90e37e82cf00.png)
![277885097_517926829916549_7868806907763248264_n](https://user-images.githubusercontent.com/103198639/162240754-6d5546ef-01f3-4efa-ba76-6275fb34cc14.jpg)
![277848521_3185952641675470_2327487045108311686_n](https://user-images.githubusercontent.com/103198639/162240770-ead86a8a-7d78-4d49-96a2-045b2c45b5b3.jpg)
![277906242_671517787389758_2566250193525985375_n](https://user-images.githubusercontent.com/103198639/162240773-7a6ef352-7cef-45a3-9c20-5a948f263b65.jpg)

**Comment by arkypita** [comment]
Settings look good to me, you can try increasing acceleration because 500 mm/sec2 is very low value, you may never reach the requested engraving speed due to acceleration/deceleration.

Keep in mind that plywood is a very unpleasant material because it does not react linearly to engraving power, i mean that the material can stay white until you reach 30-40% of power and then immediately turn black in the range 40-50%

Other possibility is that your laser module laser diode driver is not able to modulate the power well, maybe because it has some capacitor that slow down the response of modulation pulses.

Are you using dithering or line2line?

**Reply by MArTI9999** [reply]
Left picture of my dog is dithering,right line2line.
I tried with 800mm/sec acceleration but with no effect.
If its capacitor I need to open laser laser or its on cnc shield?

**Comment by arkypita** [comment]
I suggest to use this testfile ([here is a link](https://drive.google.com/file/d/11GT_Y22bOzu9UBd7_sKZAZgLAcKWkzI0/view?usp=sharing) to original uncompressed file) to test grayscale performance of laser/material pair.

![greytest](https://user-images.githubusercontent.com/8782035/162250367-c3d67a9b-e123-451d-8818-24fb79d8beb9.png)

Use "passthrough" LaserGRBL option, M3, SMin 0, SMax 1000 and use a speed that your laser is able to reach really without being limited by the acceleration (i.e put 3000 and verify that in the upper corner of lasergrbl you can see that the laser moves at F3000).

Now look at the result.

You will see some rectangle where you cannot see the difference between the circle and the rectangle filling: this is your new S-Min.
You will see some rectangle where you cannot see the difference between the rectangle filling and his border: this is your new S-Max.

Retry the engraving with this S-Min/S-Max combination and see the result.

Each rectangle is spaced 10%

**Comment by MArTI9999** [comment]
I did what you proposed.From 10% to 30% and 80% to 100% I don't see difference so I set Smin 30% and Smax 80% speed F3000 and F2000 but effect is similar to previous ones.There's still completely white spots and so dark areas that can't see details.
Also question.DPI and/or quality have meaning when I engraving small image like 3cm or less?

![277963508_399602092168219_4482958694391935536_n](https://user-images.githubusercontent.com/103198639/162262011-9cd7ace2-0075-4c3c-937f-d1f21e69bb51.jpg)

**Comment by lady-65** [comment]
my laser is not burning into wood or anything. not sure why??/ help watched all videos...

**Comment by lady-65** [comment]
24/11/2022 13:56:26.562	Program     	------- LaserGRBL v4.9.4 [64bit] START -------
24/11/2022 13:56:31.154	Update DB   	Material DB is up to date
24/11/2022 13:56:33.036	SetStatus   	Machine status [Connecting]
24/11/2022 13:56:33.043	OpenCom     	Open COM3 @ 115200 baud  (UsbSerial)
24/11/2022 13:56:33.049	SendImmediate	Send Immediate Command [0x18]
24/11/2022 13:56:33.063	SetStatus   	Machine status [Idle]
24/11/2022 13:56:33.071	VersionInfo 	Detected Grbl v1.1h
24/11/2022 13:56:33.656	EnlargeBuffer	Buffer size changed to 128 [True]
24/11/2022 13:58:14.780	OpenFile    	Open C:\Users\lt_la\Downloads\LaserGRBL-accuracy-test-file.svg
24/11/2022 14:04:47.318	SetStatus   	Machine status [Run]
24/11/2022 14:04:53.068	SetStatus   	Machine status [Idle]
24/11/2022 14:04:59.068	SetStatus   	Machine status [Run]
24/11/2022 14:05:04.567	SetStatus   	Machine status [Idle]
24/11/2022 14:05:07.779	EnqueueProgram	Push Header
24/11/2022 14:05:07.779	EnqueueProgram	Push File, 1035 lines
24/11/2022 14:05:07.780	EnqueueProgram	Running program, 1035 lines
24/11/2022 14:05:07.818	SetStatus   	Machine status [Run]
24/11/2022 14:06:09.093	ProgramEnd  	Job Executed: 1035 lines, 0 errors, 1 min,1 sec
24/11/2022 14:06:09.094	EnqueueProgram	Push Footer
24/11/2022 14:06:09.566	SetStatus   	Machine status [Idle]
24/11/2022 14:16:30.568	SetStatus   	Machine status [Run]
24/11/2022 14:16:36.316	SetStatus   	Machine status [Idle]
24/11/2022 14:16:39.567	SetStatus   	Machine status [Run]
24/11/2022 14:16:45.317	SetStatus   	Machine status [Idle]
24/11/2022 14:16:57.818	SetStatus   	Machine status [Run]
24/11/2022 14:17:03.568	SetStatus   	Machine status [Idle]
24/11/2022 14:17:09.318	SetStatus   	Machine status [Run]
24/11/2022 14:17:15.068	SetStatus   	Machine status [Idle]
24/11/2022 14:17:57.495	EnqueueProgram	Push Header
24/11/2022 14:17:57.495	EnqueueProgram	Push File, 1035 lines
24/11/2022 14:17:57.495	EnqueueProgram	Running program, 1035 lines
24/11/2022 14:17:57.568	SetStatus   	Machine status [Run]
24/11/2022 14:18:58.804	ProgramEnd  	Job Executed: 1035 lines, 0 errors, 1 min,1 sec
24/11/2022 14:18:58.804	EnqueueProgram	Push Footer
24/11/2022 14:18:59.316	SetStatus   	Machine status [Idle]
24/11/2022 14:20:17.799	OpenFile    	Open C:\Users\lt_la\Downloads\LaserGRBL-accuracy-test-file.svg
24/11/2022 14:21:50.318	SetStatus   	Machine status [Run]
24/11/2022 14:21:56.067	SetStatus   	Machine status [Idle]
24/11/2022 14:22:00.068	SetStatus   	Machine status [Run]
24/11/2022 14:22:05.818	SetStatus   	Machine status [Idle]
24/11/2022 14:25:39.569	SetStatus   	Machine status [Run]
24/11/2022 14:25:45.318	SetStatus   	Machine status [Idle]
24/11/2022 14:25:56.252	EnqueueProgram	Push Header
24/11/2022 14:25:56.252	EnqueueProgram	Push File, 1035 lines
24/11/2022 14:25:56.252	EnqueueProgram	Running program, 1035 lines
24/11/2022 14:25:56.319	SetStatus   	Machine status [Run]
24/11/2022 14:26:57.560	ProgramEnd  	Job Executed: 1035 lines, 0 errors, 1 min,1 sec
24/11/2022 14:26:57.560	EnqueueProgram	Push Footer
24/11/2022 14:26:57.570	SetStatus   	Machine status [Idle]
24/11/2022 14:26:57.816	SetStatus   	Machine status [Run]
24/11/2022 14:26:58.066	SetStatus   	Machine status [Idle]
24/11/2022 14:27:08.830	CloseCom    	Close COM3 [USER]
24/11/2022 14:27:08.928	SetStatus   	Machine status [Disconnected]
24/11/2022 14:27:09.400	Program     	---------------- PROGRAM STOP ----
